### PR TITLE
Win_hostname module: Change name check to WMI

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
Fix non netbios compliant name handling - #55283 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
win_hostname.ps1

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: changelogs/fragments/56420-Fix-win_hostname-for-non-netbios-names.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Fix non netbios compliant name handling ()
```
Comment (jborean93) [inline changelogs/fragments/56420-Fix-win_hostname-for-non-netbios-names.yml]: Probably good to include the module and issue it is referencing, e.g. `win_hostname`.

**Review by jborean93 — CHANGES_REQUESTED** [review]

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
This is a good candidate for backporting to stable-2.8.

**Comment by ShachafGoldstein** [comment]
> This is a good candidate for backporting to stable-2.8.

So simply create a new PR of this into the said branch?

**Comment by jborean93** [comment]
Have a look at https://docs.ansible.com/ansible/latest/community/development_process.html#backporting-merged-prs. It involves "cherry-picking" the commit to the older stable-2.x branch then creating a PR with that.

**Pull request merged**